torch/csrc/jit/codegen/cuda/arith.h

```diff
@@ -43,7 +43,8 @@ TORCH_CUDA_API TensorView* reductionOp(
     BinaryOpType reduction_op_type,
     const std::vector<int>& axes,
     Val* init,
-    TensorView* v1);
+    TensorView* v1,
+    c10::optional<bool> keep_dim = c10::nullopt);
```

Comment:
nitpick, `c10::optional` seems a little redundant here. c10::nullopt is equivalent to false. Maybe we can keep it simple and have a default value of false for `keep_dim`.

Reply:
Thanks for the suggestion. I have replaced `c10::optional<bool>` with `bool` for all the keep_dim's.